packages/atoms/booker/types.ts

```diff
@@ -0,0 +1,55 @@
+/**
+ *
+ * These props be used to already set the component in a specific state,
+ * even before the data is fetched. Useful for rendering this component
+ * on the server.
+ */
+export interface OptionalServersideSettings {
+  /**
+   * Sets the Booker component to the away state.
+   * This is NOT revalidated by calling the API.
+   */
+  isAway?: boolean;
+  /**
+   * If false and the current username indicates a dynamic booking,
+   * the Booker will immediately show an error.
+   * This is NOT revalidated by calling the API.
+   */
+  allowsDynamicBooking?: boolean;
+}
+
+/**
+ * @TODO: Abstract this into a reusable type? Or even create a context/zustand
+ * people can wrap so they only have to set this once? Although I doubt that
+ * people will wrap multiple components in one page and thus one context.
```

Comment:
Smart - we can do this context or zustand - i personally vote zustand as its a little lighter and will allow us to move this out of react if we ever choose to too. zustand stores don't depend on react